package.json

```diff
@@ -31,12 +31,15 @@
   },
   "devDependencies": {
     "@signalk/client": "^2.0.4",
-    "@babel/core": "^7.7.2",
-    "@babel/preset-env": "^7.7.1",
-    "@babel/register": "^7.7.0",
-    "@babel/preset-react": "^7.7.0",
+    "@babel/core": "^7.8.4",
```

Comment:
Maybe next time put updates to existing deps in a separate commit, so that the additions that go with the feature are easier to distinguish.

Reply:
I needed an upgrade so that I could transplile some jsx file (WidthProvider.jsx)